haskell/sbp.cabal

```diff
@@ -1,5 +1,5 @@
 name:                  sbp
-version:               2.2.3
+version:               2.2.3-dirty
```

Comment:
Process is usually commit changes in `.yaml`, generate code, fix tests if any and commit. The `-dirty` means you ran the generator with uncommitted changes:

```
➜  ~/repos/libsbp git:(master) git describe --tags --dirty --always
v2.2.3
➜  ~/repos/libsbp git:(master) echo woops > README.md 
➜  ~/repos/libsbp git:(master) ✗ git describe --tags --dirty --always
v2.2.3-dirty
➜  ~/repos/libsbp git:(master) ✗ 
```